internal/experiments/chaos_controller/chaosmesh.go

```diff
@@ -0,0 +1,232 @@
+package chaos_controller
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+	"xk6-khorne/internal/k8s/pods"
+	"xk6-khorne/pkg/k8s/client"
+
+	coreV1 "k8s.io/api/core/v1"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+type ChaosController struct {
+	cluster_config_path string
```

Comment:
```suggestion
	clusterConfigPath string
```
[Effective Go > Names > Mixed Caps](https://go.dev/doc/effective_go#names) reference.